### PR TITLE
Fixed watch page

### PR DESCRIPTION
**Pull request by chiiida**
Please review:
- [x] All videos work well
- [x] Responsive

### CODE REVIEW

**Comment by tiemfah** [comment]
it works!

File: next.config.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-            '/participate':{page:'/participate'},
-        };
+const withImage = require("next-images");
+const withCSS = require("@zeit/next-css");
+module.exports = withCSS(
+  withImage({
+    exportPathMap: function() {
+      return {
+        "/": { page: "/index" },
+        "/watch": { page: "/watch" },
```
Comment (tiemfah) [inline next.config.js:8]: add your page here

**Review by tiemfah — CHANGES_REQUESTED** [review]
add your page to next.config.js so next.js will render it.

File: next.config.js (hunk 1 above)
Comment (chiiida) [reply next.config.js:8]: Got it ka 👍

**Review by tiemfah — APPROVED** [review]

**Review by patdpat — APPROVED** [review]

**Comment by chiiida** [comment]
Please review the watch page:
- [x] All videos work well
- [x] Responsive

Please review sponsor page:
- [x] Accordion work
- [x] Responsive

**Review by Klan300 — APPROVED** [review]

**Review by Sirikonss — APPROVED** [review]

**Pull request merged**